client/src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers/';
+
+let store = createStore(rootReducer, applyMiddleware(thunk));
```

Comment:
Can we add in the code that enables Redux DevTools to work?

(from a different project:)
`const store = createStore(rootReducer, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());`